examples/data_collect/main.cpp

```diff
@@ -422,6 +422,18 @@ int main(int argc, char *argv[]) {
     FrameHandler frameSaver;
     frameSaver.storeFramesToSingleFile(saveToSingleFile);
 
+    //Request 2 adittional frames when measuring the network speed 
+    //to remove the delay caused by frame processing 
+    if (useNetLinkTest) {
+        for (int i = 0; i < 2; i++) {
+            status = camera->requestFrame(&frame);
+            if (status != Status::OK) {
+                LOG(ERROR) << "Could not request frame!";
+                return 0;
+            }
+        }
+    }
+
```

Comment:
I think that the best way to do this is inside server. The server should handle this kind of details.
This way viewer, any python script could benefit from this.